scripts/lib/install

```diff
@@ -52,6 +52,23 @@ Options:
   --no-dist-upgrade
       Skip the initial `apt-get dist-upgrade`.
 
+  --push-notifications
+      Enable Zulip mobile push notifications. This will “phone home”
+      to Zulip Cloud to register this server, and you'll be prompted
+      to agree to the Terms of Service.
```

Comment:
Is there a reason the language here is quite different from the installation instructions?

Reply:
Hmm not really, just written independently, trying to be concise, ending up with different phrasing. No reason not to make this as similar as possible to be fair. I tweaked this to:
```
  --push-notifications
      With this option, the Zulip installer registers your server for the Mobile
      Push Notification Service, and sets up the initial default configuration.
      You will be immediately prompted to agree to the Terms of Service, and your
      server will be registered at the end of the installation process.
```

so just removing the links compared to the installation doc and removing the "You can learn more about the service....." sentence.